gpt4all-bindings/python/makefile

```diff
@@ -13,4 +13,13 @@ wheel:
 
 clean:
 	rm -rf {.pytest_cache,env,gpt4all.egg-info}
-	find . | grep -E "(__pycache__|\.pyc|\.pyo$\)" | xargs rm -rf
\ No newline at end of file
+	find . | grep -E "(__pycache__|\.pyc|\.pyo$\)" | xargs rm -rf
+
+black:
```

Comment:
since these run in the venv perhaps make them depend on the venv target so the venv gets created when running them if it doesn't exist? (and either rename the dir or the target so that actually gets detected properly) / make sure the venv has these deps?